relay-server/src/metrics_extraction/sessions.rs

```diff
@@ -184,35 +185,17 @@ pub fn extract_session_metrics<T: SessionLike>(
         }
     }
 
-    // Count durations for all exited/crashed sessions. Note that right now, in the product we
-    // really only use durations from session.status=exited, but decided it may be worth ingesting
-    // this data in case we need it. If we need to cut cost, this is one place to start though.
+    // Count durations for all exited sessions. Note that right now, in the product we
+    // really only use durations from session.status=exited.
     if let Some((duration, status)) = session.final_duration() {
-        target.push(Metric::new_mri(
-            METRIC_NAMESPACE,
-            "duration",
-            MetricUnit::Duration(DurationUnit::Second),
-            MetricValue::Distribution(duration),
-            timestamp,
-            with_tag(&tags, "session.status", status),
-        ));
-    }
-
-    // Some SDKs only provide user information on subsequent updates (not when init==true), so
-    // collect the user ID here as well.
```

Comment:
I don't think I understand this. Is it possible for an SDK to send an errored session, and the user information of that session in a subsequent update?
- If yes, is the subsequent update marked as an errored session?
- If no, and the subsequent updates don't contain any errors, is the user counted in two metrics?

Looking at the changes above, I'm looking for an edge case of relay emitting more than one metric for a session (even if there are multiple updates) and understand the possible consequences it may have.